modules/ycc_vm.py

```diff
@@ -617,7 +617,22 @@ def delete_vm(self):
         return response
 
     def update_vm(self):
-        raise NotImplementedError('update action not implemented')
+        response = dict()
+        name = self.params.get('name')
+        folder_id = self.params.get('folder_id')
+        labels = self.params.get('labels')
+        instance = self._get_instance(name, folder_id)
+        protobuf_field_mask = FieldMask(paths=['labels'])
+        if instance:
+            self.instance_service.Update(UpdateInstanceRequest(
+                instance_id=instance['id'],
+                labels=labels,
+                update_mask=protobuf_field_mask
+            ))
+            response['msg'] = 'Updated'
+        else:
+            response['msg'] = 'No instance found'
```

Comment:
I think we should fail if instance is missing in cloud. Take a look in add action how it is done...